pkg/daemon/ceph/osd/volume.go

```diff
@@ -282,12 +279,18 @@ func (a *OsdAgent) initializeBlockPVC(context *clusterd.Context, devices *Device
 			} else {
 				deviceArg = device.Config.Name
 			}
+			var immediateExecuteArgs []string
 
-			immediateExecuteArgs := append(baseArgs, []string{
+			immediateExecuteArgs = append(baseArgs, []string{
```

Comment:
You could remove the declaration on line 282 and keep the `:=` assignment here.

Reply:
done.